src/test/java/com/ibm/stocator/fs/swift2d/TestSwiftFileSystemBlockLocation.java

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
@gilv I incorporated the testing for block locations thinking that it's a standard file system operation. I can remove it if you think it's unnecessary.